portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,18 +15,74 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import java.util.ArrayList;
+
+import com.google.sps.data.Comment;
+import com.google.gson.Gson;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns comments on portfolio */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello from Antonia!</h1>");
+
+    ArrayList<Comment> comArray = new ArrayList<Comment>();
```

Comment:
couple of things:
* I would name the list `comments` instead of `comArray`. The important part is that it contains comments, not that it is a container.
* Java allows to omit the instantiating generic, shortens the line.
* we can use `List` for the variable declaration if we don't use `ArrayList` specific functions. It's more of a nit.

So we can write: 
```suggestion
    List<Comment> comments = new ArrayList<>();
```